service/src/main/java/tds/session/repositories/impl/ExternalSessionConfigurationRepositoryImpl.java

```diff
@@ -25,15 +25,24 @@ public ExternalSessionConfigurationRepositoryImpl(final DataSource dataSource) {
         final SqlParameterSource parameters = new MapSqlParameterSource("clientName", clientName);
 
         String query = "SELECT " +
-            "clientname as clientName, \n" +
-            "environment \n" +
+            "externs.clientname as clientName, \n" +
+            "externs.environment, \n" +
+            "_externs.shiftwindowstart, \n" +
+            "_externs.shiftwindowend \n" +
             "FROM session.externs \n" +
-            "WHERE clientname = :clientName";
+            "JOIN session._externs ON _externs.clientName = externs.clientName \n" +
```

Comment:
Since this is a microservice that will only have a single database that backs it, do we need to include the name of the database in the queries or can we just have it in the data source connection and just query tables directly?